src/components/events/createEvent/basicDetails.tsx

```diff
@@ -6,12 +6,12 @@ import {
   Typography,
 } from '@mui/material';
 import SelectField from '../../general/selectField';
-import DateRangePicker from '../commonEventFields/dateRangePicker';
-import { BLOCKS, EVENT_TYPES } from '../../../types/enums';
+import DateRangePicker from '../../general/dateRangePicker';
+import { BLOCKS, EVENT_TYPES } from '../../../types/string-types';
```

Comment:
Please consider using absolute imports! Typescript actually handles it pretty good, check it out